###############################################
########## Physical robot parameters ##########
###############################################

Gofa:
  ip_robot: "192.168.131.200"
  robot_port_rws: 443
  # ip_robot: "192.168.56.101" # for virtual robot (robotstudio)
  # robot_port_rws: 80 # for virtual robot (robotstudio)
  name_robot: "ROB_1"
  task_robot: "T_ROB1"
  pos_corr_gain: 1
  max_speed_deviation: 300
  robot_port_egm: 6511
  root_name: "base"
  tip_name: "L6"

###############################################
#### Settings for ros_control control loop ####
###############################################

gofa_hw_control_loop:
  loop_hz: 250
  cycle_time_error_threshold: 0.004

###############################################
# Settings for ros_control hardware interface #
###############################################

gofa_hardware_interface:
   joints:
      - joint1
      - joint2
      - joint3
      - joint4
      - joint5
      - joint6